src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -238,6 +267,8 @@ public function output(
      *
      * @throws \RuntimeException
      *
+     * @deprecated This method is deprecated with no replacement since sonata-project/admin-bundle 3.x and will be removed in 4.0.
```

Comment:
You should add a trigger_error message in the method then.

And a `NEXT_MAJOR: Remove this method`.

Reply:
Thanks